### ISSUE TITLE
Gens upload failing silently

### ISSUE THREAD

**Issue by raysloks** [Clinical-Genomics/cg#2128]
## Description

When `cg upload` runs `gens load sample`, it fails silently when getting a `DuplicateKeyError`-exception from pymongo. This is problematic when uploading 

## Suggested solution

There's two main ways to solve this.
1. Replace old entries with new ones
2. Add an additional key to distinguish between entries

## This can be closed when

*Describe what needs to be done for this issue to be closed*

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by ChrOertlin** [comment]
@raysloks is this issue still relevant? Then I will put it up for refinement.

**Comment by raysloks** [comment]
Oh yeah, this is effectively a duplicate of https://github.com/Clinical-Genomics/cg/issues/3197. I just never got around to it the first time around.
